tools/mOTUs/.shed.yml

```diff
@@ -0,0 +1,13 @@
+name: mOTUs
+owner: iuc
```

Comment:
Was your intention to submit this tool to the IUC repo or this repo here ... both are different, both are fine.